lms/static/js/discovery/main.js

```diff
@@ -0,0 +1,22 @@
+RequireJS.require([
+    'jquery',
+    'backbone',
+    'js/discovery/app',
+    'js/discovery/collection',
+    'js/discovery/form',
+    'js/discovery/result_list_view',
+    'js/discovery/filter_bar_view',
+    'js/discovery/search_facets_view'
+], function ($, Backbone, App, Collection, DiscoveryForm, ResultListView, FilterBarView, FacetsBarView) {
```

Comment:
I'm a bit concerned about relying so heavily on RequireJS in the LMS without using RequireJS Optimizer.  In the example sandbox, I'm seeing 13 separate requests to load these resources, and the JavaScript hasn't been minimized.  Also, it looks like we're using the git commit sha as a cache-busting mechanism, which means every deploy will force clients to re-download these resources, even if they haven't changed.

Have we measured the impact on client-side performance with this change?